docs/contributors/code/react-native/integration-test-guide.md

```diff
@@ -338,7 +338,7 @@ If you have trouble locating an element’s identifier, you may wish to use Xcod
 
 ### False positives when omitting `await` before `waitFor` function
 
-Omitting the `await` before a `waitFor` can lead to scenarios where tests pass but are not testing the intended behaviour. For example, if `toBeDefined` is used to assert the result of a call to `waitFor`, the assertion will pass because `waitFor` returns a value, even though it’s not the `ReactTestInstance` we meant to check for.
+Omitting the `await` before a `waitFor` can lead to scenarios where tests pass but are not testing the intended behaviour. For example, if `toBeDefined` is used to assert the result of a call to `waitFor`, the assertion will pass because `waitFor` returns a value, even though it’s not the `ReactTestInstance` we meant to check for. For this reason, it's recommended to use the custom matcher `toBeVisible` which prevents this case.
```

Comment:
```suggestion
Omitting the `await` before a `waitFor` can lead to scenarios where tests pass but are not testing the intended behaviour. For example, if `toBeDefined` is used to assert the result of a call to `waitFor`, the assertion will pass because `waitFor` returns a value, even though it is not the `ReactTestInstance` we meant to check for. For this reason, it is recommended to use the custom matcher `toBeVisible` which guards against this type of false positive.
```

Minor grammar modifications to improve comprehensibility. Feel free to ignore if you disagree.

Reply:
Thanks for the suggestions, I've applied them in https://github.com/WordPress/gutenberg/pull/38254/commits/023e8872b2dddf296ee7fe55b68f7be82a983196.